h/services/search_index/_queue.py

```diff
@@ -1,23 +1,14 @@
 from collections import defaultdict
-from datetime import datetime, timedelta
 
 from dateutil.parser import isoparse
-from sqlalchemy import and_, func, literal_column, select
-from zope.sqlalchemy import mark_changed
 
 from h.db.types import URLSafeUUID
-from h.models import Annotation, Job
+from h.models import Annotation
 
 
 class Queue:
```

Comment:
This class is now about taking elements out of the queue to reindex them.

Reply:
Yes makes sense, like but like the service rename I'd like to do this one a separate PR after this.

Reply:
Done